app/redhat_insights/default/data/ui/views/main_dashboard.xml

```diff
@@ -1165,7 +1165,7 @@
 					"earliest": "-48h",
 					"latest": "now"
 				},
-				"query": "index=redhatinsights \n  | spath \n  | spath path=events{} output=events\n  | stats by _time, events, application, event_type, account_id, context.display_name\n  | mvexpand events \n  | eval _raw=events\n  | kv\n  | table _time\n  | timechart span=30d aligntime=latest count\n  | appendpipe [| stats count | where count=0 | addinfo | eval time=info_min_time.\" \".info_max_time | makemv time | mvexpand time | table time count | rename time as _time ]"
+				"query": "index=redhatinsights \n  | spath \n  | spath path=events{} output=events\n  | stats by _time, events, application, event_type, account_id, context.display_name\n  | mvexpand events \n  | eval _raw=events\n  | kv\n  | table _time\n  | where relative_time(now(), \"-30d\") <= _time\n  | timechart span=30d count(_time) as count\n  | appendpipe [| stats count | where count=0 | addinfo | eval time=info_min_time.\" \".info_max_time | makemv time | mvexpand time | table time count | rename time as _time ]"
```

Comment:
@fellipeh can you please explain these changes?

`queryParameters` sets it to `"earliest": "-48h"`

Reply:
This `queryParameters`  doesn't make any influence on that query, we can set anything different and still not working..